### PR TITLE
Open proper project merge

### PR DESCRIPTION
**Pull request by mdjdrn1**
Added proper opening of the project. Every project have now separate EditorModel and separate ActiveUserHandler list. 

Changed communication between EditorController and EditorModel - now it happens via ProjectManager.

### CODE REVIEW

File: src/main/java/textEditor/model/EditorModelImpl.java [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
-public class EditorModelImpl implements EditorModel, RemoteObservable {
+public class EditorModelImpl implements EditorModel, Serializable {
     private String text = "";
-    private StylesHolder styleSpans;
-
-    private ArrayList<RemoteObserver> observers;
+    private StylesHolder stylesHolder;
+    private List<RemoteObserver> observers;
 
     public EditorModelImpl() throws RemoteException {
```
Comment (GitSakro) [inline src/main/java/textEditor/model/EditorModelImpl.java:17]: This ctor is never used

File: src/main/java/textEditor/model/ProjectManagerImpl.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+            ActiveUserHandler handlerExport = (ActiveUserHandler) UnicastRemoteObject.exportObject(handler, 0);
+            registry.rebind(handlerId, handlerExport);
+            this.activeUserHandlerBinding = new Pair<String, ActiveUserHandler>(handlerId, handler);
+        }
+
+        return activeUserHandlerBinding.getKey();
+    }
+
+    // TODO: find better place for that method
```
Comment (GitSakro) [inline src/main/java/textEditor/model/ProjectManagerImpl.java:81]: I think we should delete such comment from delivery branch

**Review by GitSakro — CHANGES_REQUESTED** [review]
Delete unused ctor and TODOS

File: src/main/java/textEditor/model/ProjectManagerImpl.java (hunk 2 above)
Comment (mdjdrn1) [reply src/main/java/textEditor/model/ProjectManagerImpl.java:81]: It's removed in another PR

**Review by GitSakro — APPROVED** [review]

File: src/main/java/textEditor/model/EditorModelImpl.java (hunk 1 above)
Comment (mdjdrn1) [reply src/main/java/textEditor/model/EditorModelImpl.java:17]: I didn't add it LOL

**Pull request merged**